packages/client/lib/commands/GEORADIUSBYMEMBERSTORE.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { strict as assert } from 'assert';
+import testUtils, { GLOBAL } from '../test-utils';
+import { transformArguments, transformReply } from './GEORADIUSBYMEMBERSTORE';
+
+describe('GEORADIUSBYMEMBERSTORE', () => {
+    describe('transformArguments', () => {
+        it('STORE', () => {
+            assert.deepEqual(
+                transformArguments('key', 'member', 3 , 'm', 'dest', {
+                    SORT: 'ASC',
+                    COUNT: {
+                        value: 1,
+                        ANY: true
+                    }
+                }),
+                ['GEORADIUSBYMEMBER', 'key', 'member', '3', 'm', 'ASC', 'COUNT', '1', 'ANY', 'STORE', 'dest']
+            );
+        });
+
+        it('STOREDIST', () => {
+            assert.deepEqual(
+                transformArguments('key', 'member', 3 , 'm', 'dest', {}, true),
+                ['GEORADIUSBYMEMBER', 'key', 'member', '3', 'm', 'STOREDIST', 'dest']
+            );
+        });
+    });
+
+    it('transformReply with empty array (https://github.com/redis/redis/issues/9261)', () => {
```

Comment:
?

Reply:
I guess I took it from GEOSEARCHSTORE. Delete it?